Formula/tetra.rb

```diff
@@ -0,0 +1,21 @@
+class Tetra < Formula
+  desc "Tetragon CLI to observe, manage and troubleshoot Tetragon instances"
+  homepage "https://github.com/cilium/tetragon"
+  url "https://github.com/cilium/tetragon/archive/refs/tags/v0.8.3.tar.gz"
+  sha256 "0f8d7fea833305a20fe31374b05de858bddb1712c80823f5edc85909e6c241cd"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-s -w -X github.com/cilium/tetragon/pkg/version.Version=#{version}"
+    system "go", "build", *std_go_args(output: bin/"tetra", ldflags: ldflags), "./cmd/tetra"
+
+    generate_completions_from_executable(bin/"tetra", "completion", base_name: "tetra")
```

Comment:
that should be enough
```suggestion
    generate_completions_from_executable(bin/"tetra", "completion")
```